crates/gen-host-js/tests/helpers.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-ignore
+import { readFile } from 'node:fs/promises';
+// @ts-ignore
+import { argv, stdout } from 'node:process';
+
+// This is a helper function used from `host.ts` test in the `tests/runtime/*`
+// directory to pass as the `instantiateCore` argument to the `instantiate`
+// function generated by `wit-bindgen`.
+//
+// This function loads the module named by `path` and instantiates it with the
+// `imports` object provided. The `path` is a relative path to a wasm file
+// within the generated directory which for tests is passed as argv 2.
+export async function loadWasm(path: string, imports: any) {
```

Comment:
dumb question because I do not know Typescript: I'd expect this to have a type annotation saying it returns a `WebAssembly.Instance`, but I guess that is just an inferred type?

Reply:
I believe it's inferred, yeah, although I'm no typescript-whiz myself!